src/library/export/trackexportdlg.h

```diff
@@ -23,29 +26,42 @@ class TrackExportDlg : public QDialog, public Ui::DlgTrackExport {
 
     // The dialog is prepared, but not shown on construction.  Does not
     // take ownership of the export worker.
-    TrackExportDlg(QWidget *parent, UserSettingsPointer pConfig,
-                   TrackExportWorker* worker);
-    virtual ~TrackExportDlg() { }
+    TrackExportDlg(QWidget* parent,
+            UserSettingsPointer pConfig,
+            TrackPointerList& tracks,
+            Grantlee::Context* context = nullptr);
```

Comment:
Can we remove the default nullptr?

Reply:
Why ? You can pass a custom context if you have additional information you want to pass. There are cases now that do not have any additional context to give, so they pass nothing.